tests/conftest.py

```diff
@@ -44,4 +44,4 @@ def enable_test_mode_by_default_for_autologging_integrations():
     are raised and detected. For more information about autologging test mode, see the docstring
     for :py:func:`mlflow.utils.autologging_utils._is_testing()`.
     """
-    test_mode_on()
+    yield from test_mode_on()
```

Comment:
I'm still not confident enough but `test_mode_on()` just creates a generator and does nothing, right?

Reply:
```python
# tests/sklearn/test_sklearn_autolog.py

def test_estimator(fit_func_name):
    assert os.environ["MLFLOW_AUTOLOGGING_TESTING"] == "true"
    ...
```